tools/generate-proxies/utils/cem-helper-functions.ts

```diff
@@ -0,0 +1,93 @@
+import { execSync } from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+import { ClassMember, Package } from 'custom-elements-manifest/schema';
+import { LitCustomElement, ReactiveProperty } from './types';
+
+export function getElementMetadata(libPath: string): LitCustomElement[] {
```

Comment:
Note to self: 
These three methods (`getElementMetadata`, `createLitElementMetadata` and `getCustomElements`) definitely needs some naming work! 😅